components/IssueCredentialOptions.yml

```diff
@@ -35,7 +35,7 @@ components:
               description: The type of credential status to issue the credential with
         credentialId:
           type: string
-          description: An identifier that can be used to identify the credential. credentialId SHOULD NOT be set if the id property is set. If credential.id is set, that value will be duplicated for credentialId. If neither credentialId nor id is set, there will be no way to uniquely identify a credential or deal with duplicate errors. This property should not be used as a replacement for credential.id, rather as a means of identifying a credential without the id property set.
+          description: A URI that can be used to identify the credential, that will be used in APIs to refer to the issued verifiable credential. If not provided, the value will be auto-populated from credential.id. If credentialId is not provided and credential.id is not provided, it is not possible to refer to this credential once issued or deal with duplicate errors. credentialId SHOULD NOT be set if the credential.id property is set. This property should not be used as a replacement for credential.id, rather as a means of identifying a credential without the id property set. The issuer SHOULD NOT auto-generate a credentialId if one is not supplied because doing so could create a partitioning error if the result is never received by the client. 
```

Comment:
```suggestion
          description: A URI that can be used to identify the credential, that will be used in APIs to refer to the issued verifiable credential. If not provided, the value will be auto-populated from credential.id. If neither credentialId nor credential.id is provided, it is not possible to refer to this credential once issued nor to deal with duplicate errors. credentialId SHOULD NOT be set if the credential.id property is set. This property should not be used as a replacement for credential.id; rather, it is a means of identifying a credential without the id property being set. The issuer SHOULD NOT auto-generate a credentialId if one is not supplied, because doing so could create a partitioning error if the result is never received by the client. 
```